data/src/languages_update.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+"""languages.json update
+
+Takes languages.json file and updates the script entry for a language based on
+the files that are present in data/tsv.
+"""
+import collections
+import json
+import operator
+import os
```

Comment:
`unicodedataplus` is third-party so it goes after the typing imports but before the in-module imports.